pubsub/jetstream/metadata.go

```diff
@@ -116,5 +118,17 @@ func parseMetadata(psm pubsub.Metadata) (metadata, error) {
 		m.internalAckPolicy = nats.AckExplicitPolicy
 	}
 
+	// Explicit check to default to Single mode since this was the
+	// previous default behavior.
+	if psm.Properties[pubsub.ConcurrencyKey] != "" {
```

Comment:
We strive to make all metadata keys case-insensitive. THe way here to set a default value would be to set the default value when initializing the metadata struct, such:

Replace

```go
var m metadata
```

with:

```go
m := metadata{
    Concurrency: pubsub.Single,
}
```

This way, if the property passed by the user is empty or unset, the default value is used